stable/artifactory-ha/values.yaml

```diff
@@ -113,6 +113,20 @@ artifactory:
     # Note that by default we use appVersion to get image tag
     # version:
     pullPolicy: IfNotPresent
+
+  ## Add custom init containers
+  customInitContainers: |
+  #  - name: "custom-setup"
+  #    image: "{{ .Values.initContainerImage }}"
+  #    imagePullPolicy: "{{ .Values.artifactory.image.pullPolicy }}"
+  #    command:
+  #      - 'sh'
+  #      - '-c'
+  #      - 'touch {{ .Values.artifactory.persistence.mountPath }}/example-custom-setup'
+  #    volumeMounts:
+  #      - mountPath: "{{ .Values.artifactory.persistence.mountPath }}"
+  #        name: artifactory-volume
```

Comment:
@eldada I think value of this should be just `volume` https://github.com/jfrog/charts/blob/09878bed4b355e93b35433ecb0a712db2168c2ee/stable/artifactory-ha/templates/artifactory-primary-statefulset.yaml#L214

Reply:
Fixed